tox.ini

```diff
@@ -13,9 +13,9 @@ exclude =
     __pycache__
     .tox
     test/resources
-    test-toolkit/resources
     venv/
     docker/
+    artifacts/
```

Comment:
there is a Python file in there - does it make sense to exclude?

Reply:
It was excluded before🤷‍♀️